Apps/Encounter/src/Services/EncounterService.cs

```diff
@@ -112,7 +113,7 @@ public async Task<RequestResult<IEnumerable<EncounterModel>>> GetEncounters(stri
             }
 
             // Retrieve the phn
-            RequestResult<PatientModel> patientResult = await this.patientService.GetPatient(hdid).ConfigureAwait(true);
+            RequestResult<PatientModel> patientResult = await this.patientService.GetPatient(hdid, ct: ct);
```

Comment:
The method name `GetPatient`, no longer matches it's Async nature. suggest updating it to `GetPatientAsync` as per the current conventions being implemented...

Reply:
This may be out of scope for this ticket but please consider it at least

Reply:
That will be done another as that affects other modules.